backend/app/tests/utils/document.py

```diff
@@ -37,14 +37,14 @@ def __iter__(self):
 
     def __next__(self):
         doc_id = next(self.index)
-        args = str(doc_id).split("-")
-        fname = Path("/", *args).with_suffix(".xyz")
+        key = f"{self.owner_id}/{doc_id}.xyz"
```

Comment:
`.xyz` ? even if its a dummy doc it should have valid extension for ease of understanding